### DISCUSSION TITLE
スーパーユーザ?スーパーユーザー?

### DISCUSSION THREAD

**Discussion by koizumistr** [pgsql-jp/jpug-doc#2491] [NGとする語]
#2488 をやっていて思ったのですが、superuserは「スーパーユーザ」（最後は伸ばさない）が多数派なのですが、（「スーパー」が付かない）userは「ユーザー」とやるっぽいです（と機械翻訳が提案してきました）。

（スーパーユーザは確立した単語なのだから）これはこれであり、とも思いますが、userはuserなんだから揃えるべきだ、という考え方もあるかな、と思います。

**Comment by KenichiroTanaka** [comment]
私はずっとユーザ派でしたが近年ユーザーが主流になってきたのでそちらに合わせねば、と思っていたところでした。
スーパーユーザも昔のものですので以降は、もしくは一括変換でスーパーユーザーにした方が統一感があって良いのでは無いかと考えています。

**Comment by koyhoge** [comment]
内閣告示では、原則として長音記号を最後につける方針ですね。
https://www.bunka.go.jp/kokugo_nihongo/sisaku/joho/joho/kijun/naikaku/gairai/honbun06.html

**Reply by koizumistr** [reply]
引用元には

> ただし,慣用に応じて「ー」を省くことができる。

とありまして、そこが難しいところかと。
